src/theme/css/wiki-themes/discord-messages.css

```
@@ -125,13 +139,28 @@
     position: relative;
 }
 
+.discord-message .discord-mention .discord-mention-icon {
+  width: 1em !important;
+  height: 1em !important;
+  left: 0.125em !important;
+  top: 0.125em !important;
+}
+
+.discord-message .discord-mention.discord-channel-mention {
+  padding-left: 1.2em !important;
+}
```

Inconsistent indents
```suggestion
.discord-message .discord-mention .discord-mention-icon {
    width: 1em !important;
    height: 1em !important;
    left: 0.125em !important;
    top: 0.125em !important;
}

.discord-message .discord-mention.discord-channel-mention {
    padding-left: 1.2em !important;
}
```